plato/draw/fresnel/Box.py

```diff
@@ -3,5 +3,15 @@
 from ..internal import ShapeDecorator
 
 @ShapeDecorator
-class Box(draw.Box, Lines):
+class Box(draw.Box):
     __doc__ = draw.Box.__doc__
+
+    def __init__(self, *args, **kwargs):
+        self._lines_prim = Lines()
+        super().__init__(*args, **kwargs)
+
+    def render(self, *args, **kwargs):
+        coordinates = self._get_coordinates()
+        self._lines_prim.start_points = coordinates.start_points
+        self._lines_prim.end_points = coordinates.end_points
+        return self._lines_prim.render(*args, **kwargs)
```

Comment:
We need to pass along widths and colors in the `fresnel` backend.
![image](https://user-images.githubusercontent.com/3943761/64751158-5466c300-d4e9-11e9-8cf6-713179167c1e.png)